airflow/dags/rxnorm_historical/dag_tasks.py

```diff
@@ -1,4 +1,5 @@
 from airflow.decorators import task
+from airflow.hooks.subprocess import SubprocessHook
```

Comment:
Unnecessary

Reply:
agreed - just forgot to clean that up. It is removed now